docs/content/deployments/server/deploying-on-microtica.md

```diff
@@ -0,0 +1,112 @@
+---
+description: 'Learn step-by-step.'
+addHowToData: true
+---
+
+# Deploy Your Medusa Backend on Microtica
+
+In this document, you'll learn how to deploy your Medusa backend on your AWS account with Microtica. 
+
+[Microtica](https://microtica.com) is a cloud-native delivery platform that enables you to deploy infrastructure and applications to your AWS account, while providing actionable insights to help you optimize resources, performance, & costs. 
+
+Alternatively, you can use this button to deploy the Medusa backend to Microtica directly:
+
+<a href="https://app.microtica.com/templates/new?template=https%3A%2F%2Fraw.githubusercontent.com%2Fmicrotica%2Ftemplates%2Fmaster%2Fmedusa-server%2F.microtica%2Ftemplate.yaml&utm_source=medusa&utm_medium=docs&utm_campaign=medusa" className="img-url">
+  <img src="https://microtica.s3.eu-central-1.amazonaws.com/assets/templates/logos/deploy-with-microtica.svg" alt="Deploy with Microtica" className="no-zoom-img" />
+</a>
+
+:::note
+
+The Medusa infrastructure will be provisioned on your own AWS account.
+You retain full control over your infrastructure and data while getting all the benefits of infrastructure automation.
+
+:::
+
+## Prerequisites
+
+### Medusa Backend
+
+It is assumed that you already have a Medusa backend installed locally. If you don’t, please follow the [quickstart guide](../../development/backend/install.mdx).
+
+Furthermore, your Medusa backend should be configured to work with PostgreSQL and Redis. You can follow the [Configure your Backend documentation](../../development/backend/configurations.md) to learn how to do that.
+
+### Needed Accounts
+
+- A [Microtica](https://app.microtica.com/) account. Microtica provides a free plan that you can use.
+- An [AWS](https://aws.amazon.com/) account that you’ll connect to Microtica to provision the infrastructure.
+- A [GitHub](https://github.com/) account to create a repository to host your backend's codebase.
+
+### Required Tools
+
+- Git’s CLI tool. You can follow [this documentation to learn how to install it for your operating system](../../development/backend/prepare-environment.mdx#git).
+
+---
+
+## What will be provisioned on AWS
+Since Microtica deploys on your cloud account, here are the resources that the platform is going to provision in the environment.
+- VPC, subnets and networking
```

Comment:
```suggestion
## What will be provisioned on AWS

Since Microtica deploys on your cloud account, here are the resources that the platform is going to provision in the environment.

- VPC, subnets and networking
```